mixer-completion/cmd/completion.go

```diff
@@ -29,7 +29,6 @@ type completionCmdFlags struct {
 
 var completionFlags completionCmdFlags
 
-
```

Comment:
unrelate change?

Reply:
Correct. Left over from a un-linted commit earlier. (I think mine)

Reply:
(AKA, what's on Master right now doesn't pass linting :P)

Reply:
but we lint in travis

Reply:
If I'm understanding correctly, this is left over from my hasty `--path` addition to mixer-completion. It's just a stray newline that linting wants removed.

Reply:
As in, I believe I forgot to `make lint`, and because travis takes 8 years to run, we merged in a hurry to get the release out.

Reply:
> travis takes 8 years to run, we merged in a hurry

:disappointed:

Reply:
Huh, weird..I don't merge things that haven't passed Travis... I think it passed and then you had to force push an update which maybe broke linting? And we didn't wait for that one to pass again?

Reply:
@tmarcu -- probably. I'm 99% sure it was a mistake I made either way. Luckily it's just a bonus newline.